src/DataFixtures/ActivityFixtures.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Activity;
+use App\Entity\Technology;
+use App\Entity\Type;
+use App\Entity\User;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory;
+
+class ActivityFixtures extends Fixture implements DependentFixtureInterface
+{
+    public function load(ObjectManager $manager): void
+    {
+        $faker = Factory::create();
+        for ($i = 0; $i < 10; $i++) {
+            $activity = new Activity();
+            $activity->setName($faker->jobTitle);
+            $activity->setDescription($faker->sentence);
+            $activity->setApplicationDeadline($faker->dateTimeInInterval('now', '+10 days'));
+            $activity->setFinalDeadline($faker->dateTimeInInterval('+10 days', '+30 days'));
+            $activity->setStatus($activity->getAllStatuses()[rand(0, 2)]);
```

Comment:
You have already used `array_rand()` below. Why not use it here in the same time?